spec/rollbar/middleware/js_spec.rb

```diff
@@ -106,6 +106,7 @@
 
         before do
           Object.const_set('SecureHeaders', Module.new)
+          SecureHeaders.const_set('VERSION', '3.0.0')
```

Comment:
Ah ok, I see we had `allow(SecureHeaders).to receive(:content_security_policy_script_nonce) { 'lorem-ipsum-nonce' }`. Nice